### PR TITLE
fix: header is null

### PR DESCRIPTION
**Pull request by arteevraina**
### Description :

Handled the case for the header being null.

### Please ensure Checkmark the following :

* [x] - Tested Localy
* [x] - Checked in IE11 and Firefox
* [ ] - Checked in mobile
* [ ] - Backend work
* [x] - Frontend work
* [x] - Bug Fixes
* [ ] - Enhancements

Please Review this @hsluoyz :)

### CODE REVIEW

**Comment by hsluoyz** [comment]
@BetaCat0 @438561537 please review it.

File: web/src/PolicyTable.js [hunk 1]
```diff
@@ -88,24 +88,26 @@ class PolicyTable extends React.Component {
...
-                this.updateField(index, keys[i], e.target.value);
-              }} />
-            )
-          }
-        },
-      );
-    });
-
+    
+    if(this.props.headers != null) {
```
Comment (nodece) [inline web/src/PolicyTable.js:92]: I think use `if (this.props.headers)` is better, it is null or undefined.

**Review by nodece — APPROVED** [review]

File: web/src/PolicyTable.js (hunk 1 above)
Comment (arteevraina) [reply web/src/PolicyTable.js:92]: sure thing!

**Comment by arteevraina** [comment]
@nodece Thanks for the review. Made the required changes. :)

**Review by nodece — APPROVED** [review]

**Pull request merged**